packages/hardhat-vyper/src/downloader.ts

```diff
@@ -73,7 +61,7 @@ export class CompilerDownloader {
   }
 
   public async initCompilersList(
-    { forceDownload } = { forceDownload: true }
+    { forceDownload } = { forceDownload: false }
```

Comment:
This was redownloading the list every time, which is not necessary as it's already redownlaoded when a release is not found.